cli/testdata/wallet1_solo.json

```diff
@@ -65,6 +65,7 @@
       "key": "6PYSATFztBa3CHjSR6sLAKungUEAbQUCVE16KzmaQQ38gLeYGZ9Knd5mGv",
       "label": "verify",
       "contract": {
+        "script": "VwEAEUBXAANA",
```

Comment:
?

Reply:
Contract script. I didn't touch this code, we had our wallets improperly generated:
https://github.com/nspcc-dev/neo-go/blob/24041451335d2f6ff64b8e334401d2dff7c3b211/pkg/wallet/regenerate_test.go#L239-L242

Reply:
Really strange one. Suppose I have a 10K NEF...

Reply:
Should we fix it? Because it's not only for tests, see 0b26b462:
https://github.com/nspcc-dev/neo-go/blob/8d67f17943553740fe4caa31c3e9efcf7921aba2/cli/wallet/wallet.go#L654

Reply:
It goes back to cdfb2429f279082ea5f5756dd2bdecc15b264dac, somewhere in-between Legacy (with static contract hashes that directly depend on script) and N3. It wasn't correct even back then, because of https://github.com/neo-project/proposals/blob/master/nep-6.mediawiki#contract, but now it's even more incorrect.